advance-exercises/src/components/04-OptimizeRender/OptimizeRender.js

```diff
@@ -23,11 +23,18 @@ class ListView extends React.Component {
     rowHeight: PropTypes.number.isRequired,
     renderRowAtIndex: PropTypes.func.isRequired,
   };
-
+  componentDidMount() {
+    document.getElementById('content').addEventListener('scroll', this.handleScroll);
+    this.setState({ ...this.state, numRows: Math.floor(document.getElementById('content').clientHeight / this.props.rowHeight) })
+  }
+  handleScroll = () => {
+    const topPosition= document.getElementById('content').scrollTop/30;
```

Comment:
In `.scrolTop/30;`, assigning 30 with a proper name and using it may better convey the intent of the division.